README.md

```diff
@@ -30,3 +30,88 @@ The aims of the project are:
 3.	Calculate a ratio between public and private transport journey times 
 4.	Calculate an overall metric for the entire city
 5.	Compare the results for each city
+
+### Using VS code
+
+#### Create new branch 
+
+Initially created a branch on GitHub from the develop branch. 
+Within the terminal:
+```python 
+git fetch -p
+``` 
+This will update your local branches in line with remote branches
+Within VS code you can switch branches using the button in the bottom left corner. 
+Within the terminal you can switch branches using:
+```python
+git checkout branch_name
+```
+
+#### Moving changes from one branch to another
+It is important to ensure that you are in the correct branch before making changes however, you can move changes if needed. 
+The changes need to be added but not yet committed.
+To add the changes:
+```python
+git add file_name
+```
+If you have already committed the changes you'll need to remove the last commit from the current branch:
+```python
+git reset --soft HEAD^
+```
+The changes can then be moved using the following code:
+```python
+git stash 
+git checkout correct_branch
+git stash pop 
+```
+The changes can then be added, committed and pushed to the correct branch. 
+
+### Virtual environment 
+It is recommended that a virtual environment is used to avoid dependency issues. A virtual environment can be created by following the instructions below and the environment yaml can be used to install the required versions of packages. 
+
+#### Install Miniconda
+Conda is the recommended for environment management. Both miniconda and and the full Anaconda can be used however, the following instructions is based on using miniconda. 
+Download miniconda from this link (https://docs.conda.io/en/latest/miniconda.html). Select the correct installer for your OS and follow the instructions.
+For Mac, go to terminal and run bash download_file_path (drag and drop the downloaded file into the terminal). 
+Follow the instructions within the terminal and it should download correctly. 
+
+#### Create environment
+Use Python version 3.10
+Within the command line:
+```python
+conda create -n transp_eff_310 python=3.10
+```
+transp_eff_310 was chosen for consistency across developers.
+
+#### Activate environment
+Having created the environment, the command line will give you instructions. 
+This includes how to activate and deactivate the environment:
+```python 
+conda activate transp_eff_310
+```
+
+#### Install dependencies
+To install the dependencies you must be within the project directory. If not this can be done by running the following within the terminal:
```

Comment:
This is not actually true. I used to think this too, but actually you only need the virtual environment activated.